### ISSUE TITLE
Poloniex breaking when placing orders

### ISSUE THREAD

**Issue by caioaao** [ccxt/ccxt#1069]
In version `1.10.561`, here's what I get:

```
  File "/usr/local/lib/python2.7/dist-packages/ccxt/base/exchange.py", line 1009, in create_limit_buy_order
    return self.create_order(symbol, 'limit', 'buy', *args)
  File "/usr/local/lib/python2.7/dist-packages/ccxt/poloniex.py", line 565, in create_order
    'amount': self.amount_to_precision(symbol, amount),
  File "/usr/local/lib/python2.7/dist-packages/ccxt/base/exchange.py", line 780, in amount_to_precision
    return self.truncate(amount, self.markets[symbol]['precision']['amount'])
  File "/usr/local/lib/python2.7/dist-packages/ccxt/base/exchange.py", line 436, in truncate
    return '{0:f}'.format(Decimal(num).quantize(math.pow(10, -precision)))
  File "/usr/lib/python2.7/decimal.py", line 2426, in quantize
    exp = _convert_other(exp, raiseit=True)
  File "/usr/lib/python2.7/decimal.py", line 5852, in _convert_other
    raise TypeError("Unable to convert %s to Decimal" % other)
TypeError: Unable to convert 1e-08 to Decimal
```

looks like this is the change that broke it: https://github.com/ccxt/ccxt/commit/b2163bb9f778d59dffb0b8406f6ec96a10c61463

**Comment by kroitor** [comment]
Thx! Fixing it now.

**Comment by kroitor** [comment]
This is fixed in 1.10.564. Thx again!
